lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -48,9 +55,27 @@ def method_missing(meth, *args)
 
       return mod.send(meth.to_s, *args)
     end
+
+    action = meth.to_s.delete_prefix('cmd_')
+    if mod && mod.kind_of?(Msf::Module::HasActions) && mod.actions.map(&:name).map(&:downcase).include?(action)
+       do_action(action, *args)
+    end
+
     return
   end
 
+  #
+  #
+  # Execute the module with a set action
+  #
+  def do_action(meth, *args)
+    action = mod.actions.select { |action| action.name.downcase == meth.downcase }.first
```

Comment:
Same thing here, and we can avoid unnecessarily creating and populating an extra `Array` object.

```suggestion
    action = mod.actions.find { |action| action.casecmp?(meth) }
```